### PR TITLE
Fix anti-adblock on aternos.org (Solution 1)

### PR DESCRIPTION
**Pull request by llacb47**
https://github.com/NanoMeow/QuickReports/issues/1643#issuecomment-566691583

This PR fixes anti-adblock on `aternos.org`. If this solution is not preferable, I have also created a PR (https://github.com/uBlockOrigin/uAssets/pull/6697) that fixes the anti-adblock by keeping the existing filters and whitelisting another script.

IMO this solution is preferable but it is easy for the website to break the cosmetic filter.

### CODE REVIEW

File: filters/filters.txt [hunk 1]
```diff
@@ -4593,12 +4593,9 @@ themeslide.com##+js(nano-setInterval-booster.js, countdown, 1500)
...
 ! https://github.com/NanoAdblocker/NanoFilters/issues/272
 ! https://github.com/NanoMeow/QuickReports/issues/25
-*$script,domain=aternos.org,3p,redirect-rule=noopjs
-@@||aternos.org^$generichide
-@@||amazon-adsystem.com/*/apstag.js$script,domain=aternos.org
-@@||snigelweb-com.videoplayerhub.com/videoloader.js$script,domain=aternos.org
-@@||static.h-bid.com^$css,script,domain=aternos.org
-@@*$xhr,3p,domain=aternos.org
+aternos.org##.ad
+aternos.org##+js(acis,$)
```
Comment (okiehsch) [inline filters/filters.txt]: Can this be made more specific by adding a second argument?
The second argument can be any string of the inline script you want to abort.
https://github.com/gorhill/uBlock/wiki/Resources-Library#acisjs-

**Comment by llacb47** [comment]
I'm hesitant to make `acis` specific to an inline-script containing a certain string since the site has broken our filters in the past. If they do it again, I think it should be reconsidered. There is only one inline-script that uses `$` anyway.

**Comment by okiehsch** [comment]
Thx.

**Comment by llacb47** [comment]
@okiehsch they broke the `acis` scriptlet. Can you change the second argument to `css` or remove it entirely? I am not able to make a PR right now.

**Comment by llacb47** [comment]
Thanks.

**Pull request merged**